src/__tests__/app.int-spec.ts

```diff
@@ -0,0 +1,80 @@
+import request from 'supertest';
+import * as faker from 'faker';
+import { HttpStatus } from '@shared/enums/http-status.enum';
+import { App } from '../app';
+import { aUUID, aISODate } from './helpers/matchers.helper';
+
+jest.mock('../shared/infra/typeorm');
+
+describe('App Integrations tests', () => {
+	let app: App;
+	beforeAll(async () => {
+		app = new App();
+
+		await app.init();
+		await app.listen();
+	});
+
+	it('should be defined', () => {
+		expect(app).toBeDefined();
+	});
+
+	it('should be health', () => {
+		return request(app.getServer())
+			.get('/')
+			.expect('Content-Type', /json/)
+			.expect(({ body }) => {
+				expect(body).toMatchObject({
+					health: true,
+				});
+			});
+	});
+
+	describe('Users', () => {
+		describe('POST /users - Create User', () => {
+			it('should able to create user', () => {
+				return request(app.getServer())
+					.post('/users')
+					.send({
+						name: 'Dante',
+						email: 'naguadtl@gmail.com',
+						password: 'dante',
+					})
+					.expect('Content-Type', /json/)
+					.expect(HttpStatus.CREATED)
+					.expect(({ body }) => {
```

Comment:
@stevescruz The code identation looks weird, we are using 2 for `indent_size` in `.editorconfig` and we are using tabs for identation. Maybe we can use spaces?

Reply:
Yeah, it looks really weird on your example.

We can use spaces, no problem